.github/workflows/evalml_main_performance_tests.yaml

```diff
@@ -0,0 +1,139 @@
+      - name: Await results
+        id: results
+        run: |
+          for id in ${{ steps.current_default.outputs.job_id }} ${{ steps.current_iterative.outputs.job_id }} ${{ steps.previous_default.outputs.job_id }} ${{ steps.previous_iterative.outputs.job_id }}; do
+            echo "Waiting for job id: $id"
+            result=''
+            sleep_time=0
+            while [ -z "$result"  ]
+            do
+              sleep $sleep_time
+              result=$(looking-glass get-job --job-id $id | grep -n 'COMPLETED' || :;)
+              sleep_time=60
+            done
+            looking-glass get-results --job-id $id --file-path $id
+          done
+        shell: bash
+      - name: Generate reports
+        run: |
+          filename_base=${{ steps.get_hashes.outputs.current_hash }}_v_${{ steps.get_hashes.outputs.previous_hash }}
+          looking-glass get-report --previous-path ${{ steps.previous_default.outputs.job_id }} -n ${{ steps.current_default.outputs.job_id }} --output-name ${filename_base}_default --output-type html
+          looking-glass get-report --previous-path ${{ steps.previous_iterative.outputs.job_id }} -n ${{ steps.current_iterative.outputs.job_id }} --output-name ${filename_base}_iterative --output-type html
+      - name: Upload reports to S3
+        id: s3_upload
+        run: |
+          filename_base=${{ steps.get_hashes.outputs.current_hash }}_v_${{ steps.get_hashes.outputs.previous_hash }}
+          default_url=s3://evalml-main-looking-glass-reports/${filename_base}_default.html
+          aws s3 cp ./${filename_base}_default.html $default_url
+          echo "::set-output name=default_report_url::$default_url"
+          echo "Default Algorith Report: $default_url"
+          iterative_url=s3://evalml-main-looking-glass-reports/${filename_base}_iterative.html
+          aws s3 cp ./${filename_base}_iterative.html $iterative_url
+          echo "::set-output name=iterative_report_url::$iterative_url"
+          echo "Iterative Algorith Report: $iterative_url"
+      - name: Notify on Slack
+        run: |
+          presigned_default_url=$( aws s3 presign ${{ steps.s3_upload.outputs.default_report_url }} )
+          presigned_iterative_url=$( aws s3 presign ${{ steps.s3_upload.outputs.iterative_report_url }} )
+          curl -X POST ${{ secrets.LG_SLACK_NOTIFICATION_WEBHOOK }} -H 'Content-type: application/json' \
+          --data-binary @- << EOF
+          {
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "<!subteam^${{ secrets.LG_SLACK_NOTIFICATION_USER_GROUP }}>, performance tests for latest commit on evalml are complete"
+                }
+              },
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "Author: ${{ github.event.head_commit.author.name }}"
```

Comment:
anyway to tag the author here? Does adding `@` work?

Reply:
Tag the author in slack? No. We are not consistent in our usernames/emails in git, so we'd have to maintain a mapping somewhere and hope people don't change it.

Reply:
Gotcha! Maybe we can consider doing if we think its worth it.